app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -12,17 +12,14 @@
   <%= link_to 'Export to EndNote', polymorphic_path([main_app, @presenter], format: 'endnote'), class: 'btn btn-default' %>
   <%= link_to 'Export to Zotero', sufia.static_path('zotero'), id: 'zoteroLink', name: 'zotero', class: 'btn btn-default' %>
   <%= link_to 'Export to Mendeley', sufia.static_path('mendeley'), id: 'mendeleyLink', name: 'mendeley', class: 'btn btn-default' %>
-
-  <% if @presenter.display_feature_link? %>
-    <%= link_to "Feature", sufia.featured_work_path(@presenter, format: :json),
-      data: {behavior: 'feature'}, id: 'featureLink', name: 'featureLink', class: 'btn btn-default' %>
+  <% if @presenter.respond_to?(:display_feature_link?) && @presenter.display_feature_link? %>
```

Comment:
@mjgiarlo , ok that's understandable, but at the same time thinking about the simplicity, it might be good to keep it simple for now and when future adopters might need to update, that can be done later